yt/visualization/plot_window.py

```diff
@@ -372,6 +388,31 @@ def _recreate_frb(self):
         for key in self.override_fields:
             self._frb[key]
 
+    @property
+    def _has_swapped_axes(self):
+        # note: we always run the validations here always in case the states of
+        # the conflicting attributes have changed.
+        return self._validate_swap_axes(self._swap_axes_input)
+
+    @invalidate_data
+    def swap_axes(self):
+        # toggles the swap_axes behavior
+        new_swap_value = not self._swap_axes_input
+        # note: we also validate here to catch invalid states immediately, even
+        # though we validate on accessing the attribute in `_has_swapped_axes`.
+        self._swap_axes_input = self._validate_swap_axes(new_swap_value)
+
```

Comment:
```suggestion
        self._swap_axes_input = self._validate_swap_axes(new_swap_value)
        return self

```

Reply:
thanks for catching that! these all return `self` now.